src/osd/OSD.h

```diff
@@ -1582,9 +1582,9 @@ class OSD : public Dispatcher,
 
 private:
   /**
-   *  @defgroup monc helpers
-   *
-   *  Right now we only have the one
+   * \defgroup monc helpers
```

Comment:
@tchaikov 

here:
https://www.stack.nl/~dimitri/doxygen/manual/grouping.html

it says it should be \defgroup?